kontakty_konwert.cpp

```diff
@@ -160,7 +160,7 @@ int main()
     {
         do
         {
-            getline(ff_in, linn, '\n');   //odczytuj� jedn� lini� w pliku wej�ciowym
+            getline(ff_in, linn, '\n');   // read one line in input file (odczytuj� jedn� lini� w pliku wej�ciowym)
```

Comment:
Łukasz ale to jest to niepoprawne, weź to zmień.

Reply:
na co